### DISCUSSION TITLE
Where is the config.json?

### DISCUSSION THREAD

**Discussion by Midnightzzz** [rdeepak2002/reddit-place-script-2022#381] [Q&A] [answered]
I dont see it in the list of files?

**Comment by bjb28** [comment, marked as answer]
You need to make the `config.json` based on the `config_example.json`.

**Reply by Midnightzzz** [reply]
Thanks, do you know how to fix an error (__main__:task:492  Failed to connect to websocket, trying again in 30 seconds...)

**Reply by bjb28** [reply]
Try doing at `git pull` as that could be one of the bugs just recently fixed.

**Reply by Midnightzzz** [reply]
How do I do that?

**Reply by Risewill23** [reply]
you could also just redownload the files

**Reply by Midnightzzz** [reply]
okay thanks
